frontend/src/views/Home.vue

```diff
@@ -60,10 +65,56 @@ export default {
       submitting: false,
       passwordVisible: false,
       displayError: false,
+      displayWarning: false,
+      warningMessage: '',
       errorMessage: '',
       additionalInfo: '',
     }
   },
+
+  created() {
+    var org_details = keycloak.tokenParsed.hasOwnProperty('org_details')
+    var old_ldap_id = keycloak.tokenParsed.hasOwnProperty('old_ldap_id')
+    var roleExist = false
+    var clientRole = ''
+
+    // Check if LDAP account has already been transferred and old uid exists in keycloak
+    if (old_ldap_id) {
+      this.$router.push({
+        name: 'Notification',
+      })
+      return
+    }
+    // Check if audience exists
+    if (keycloak.tokenParsed.hasOwnProperty('aud')) {
+      const aud = JSON.parse(JSON.stringify(keycloak.tokenParsed.aud))
+      for (let i = 0; i < aud.length; i++) {
+        if (aud[i].startsWith('MSPDIRECT')) {
+          clientRole = aud[i]
```

Comment:
I don't think clientRole is the right terminology here. We're trying to find the name of the audience for MSPDIRECT-SERVICE (whether it's MSPDIRECT-SERVICE or MSPDIRECT-SERVICE-UAT). audience would be a better name.